ui/src/app/pages/runtimes-list.tsx

```diff
@@ -103,8 +103,6 @@ const css =
         display: none !important
     }`;
 
-const stringToSlug = (s) => s.toLowerCase().replace(/\s+/g, '');
```

Comment:
this logic was based on two incorrect assumptions:
1. we don't have access to the full Workspace object (we do!)
2. this method always works to translate an AoU name to a Terra name.

We do use this naming algorithm when creating a workspace, but when users modify the workspace name (AoU name), the Terra name does not change, so it no longer matches this pattern.